orca/orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/spring/EchoNotifyingExecutionListener.groovy

```diff
@@ -62,14 +62,14 @@ class EchoNotifyingExecutionListener implements ExecutionListener {
         }
         AuthenticatedRequest.allowAnonymous({
           Retrofit2SyncCall.execute(
-            echoService.recordEvent(
+            echoService.recordEvent([
               details: [
                 source     : "orca",
                 type       : "orca:${execution.type}:starting".toString(),
                 application: execution.application,
               ],
               content: buildContent(execution)
-            )
+            ] as Map)
```

Comment:
Can you explain this a bit more?  What was happening before this change...some kind of magic groovy literal Map?  Why is the cast required?

Reply:
`Map<String, ?>` accepts any type for the values but when I changed it to `Map<String, Object>`, it expects all values to be exactly Object, but Groovy sometimes infers more specific types (like Map<String, String> for details). So cast is required.